tests/integrations/mcs/resourcemanager/resource_manager_test.go

```diff
@@ -1210,47 +1236,52 @@ func (suite *resourceManagerClientTestSuite) TestCheckBackgroundJobs() {
 	re := suite.Require()
 	cli := suite.client
 
-	for _, group := range suite.initGroups {
-		resp, err := cli.AddResourceGroup(suite.ctx, group)
-		re.NoError(err)
-		re.Contains(resp, "Success!")
+	enableBackgroundGroup := func(enable bool) string {
+		if enable {
+			return "background_enable"
+		} else {
+			return "background_unable"
+		}
 	}
```

Comment:
used `enableBackgroundGroup` instead of unnecessary using `initGroups` cc @nolouch